lib/bolt/config.rb

```diff
@@ -166,6 +167,12 @@ def update_from_file(data)
           self[:transports][:pcp][:orch_task_environment] = data['pcp']['task-environment']
         end
       end
+
+      if data['local']
```

Comment:
When https://github.com/puppetlabs/bolt/pull/323 is merged I want to update the test to cover this code.